src/iOS.Autofill/LoginPasswordViewController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+using Foundation;
+using UIKit;
+
+namespace Bit.iOS.Autofill
+{
+	public partial class LoginPasswordViewController : Core.Controllers.LoginPasswordViewController
```

Comment:
indenting seems off in this file

Reply:
Updated.

Reply:
Still looks off. Looks like it is using tabs or something.